hadoop-ozone/integration-test-s3/src/test/java/org/apache/hadoop/ozone/s3/awssdk/v1/AbstractS3SDKV1Tests.java

```diff
@@ -1384,6 +1387,86 @@ public void testPresignedUrlDelete() throws IOException {
     }
   }
 
+  @Nested
+  @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+  class BucketOwnershipLinkBucketTests {
```

Comment:
Maybe we could keep it consistent with [the V2 class](https://github.com/apache/ozone/blob/2029b59898a737a9c5d4a1ee54195fa16e7ed369/hadoop-ozone/integration-test-s3/src/test/java/org/apache/hadoop/ozone/s3/awssdk/v2/AbstractS3SDKV2Tests.java#L1673)